src/content/self-hosted/install/releases.mdx

```diff
@@ -6,11 +6,29 @@ id: releases
 ---
 
 ## 1.11.0
-TBD
+2 August 2023
+
+### Before you upgrade
+
+Okteto 1.11 enables by default ingress-nginx validation webhook (adding an additional Kubernetes deployment), which protects the platform against single user ingress syntax errors. <!-- 6706 -->
+
+### Features
+
+- Upgrade buildkit v0.11.6. <!-- 6771 -->
+- Upgrade ingress-nginx to 4.7.1. <!-- 6838 -->
 
 ### Improvements
 
-- Added [configuration setting in the daemonset](self-hosted/administration/configuration.mdx#daemonset) to disable the configuration of private registry credentials in the nodes <!-- Ticket: 6741, PR: 6742 -->
+- Underlaying Kubernetes cluster version is now shown in the Admin section. <!-- 6734 -->
+- A warning message is now shown at the components' logs and when attempting to authenticate or perform a GitHub related action when GitHub App private key is not valid. <!-- 6750 -->
+- Okteto Web UI documentation links now point to the same version used by the platform instead of latest. <!-- 6761 -->
```

Comment:
```suggestion
- Okteto Web UI documentation links now point to the same version used by the platform instead of the latest version. <!-- 6761 -->
```